src/lib/server/index.js

```diff
@@ -8,7 +8,9 @@ function server(site) {
   const SITE_URL = process.env[`${SITE}URL`] || config.URL;
   const SITE_DEFAULT_LOCALE =
     process.env[`${SITE}DEFAULT_LOCALE`] || config.DEFAULT_LOCALE;
-  const SITE_LOCALES_STRING = process.env[`${SITE}LOCALES`];
+  // process.env[KEY] may return empty string instead of undefined
+  const SITE_LOCALES_STRING =
+    process.env[`${SITE}LOCALES`]?.trim() || undefined;
```

Comment:
How did these changes in https://github.com/CodeForAfrica/PromiseTracker/pull/276 end up here? It hasn't been merged yet.

Reply:
@kilemensi Added this to test out the change because of the `promiseStatus.find` error?

Reply:
Nah, it's 💯 cool... I've already merged the other PR anyways.